MatrixSDK/Crypto/Data/Store/MXRealmCryptoStore/MXRealmCryptoStore.m

```diff
@@ -2165,6 +2168,20 @@ + (BOOL)finaliseMigrationWith:(RLMMigration *)migration oldSchemaVersion:(uint64
                 
             case 15:
                 MXLogDebug(@"[MXRealmCryptoStore] Migration from schema #15 -> #16: Nothing to do (added optional MXRealmSecret.encryptedSecret)");
+                
+            case 16:
+                MXLogDebug(@"[MXRealmCryptoStore] Migration from schema #16 -> #17");
+                
+                MXLogDebug(@"[MXRealmCryptoStore]    Make sure MXRealmOlmAccount.cryptoVersion is MXCryptoVersion2");
+                [migration enumerateObjects:MXRealmOlmAccount.className block:^(RLMObject *oldObject, RLMObject *newObject) {
+                    NSNumber *version;
+                    MXJSONModelSetNumber(version, oldObject[@"cryptoVersion"]);
+                    if (version && version.intValue == 0)
+                    {
+                        MXLogDebug(@"[MXRealmCryptoStore]    -> Fix MXRealmOlmAccount.cryptoVersion");
+                        newObject[@"cryptoVersion"] = @(MXCryptoVersion2);
```

Comment:
```suggestion
                        newObject[@"cryptoVersion"] = @(MXCryptoVersionLast);
```

Reply:
`MXCryptoVersionLast` makes more sense to me to use here.

Reply:
We need an accurate value to do the correct migration steps.

When we will move to MXCryptoVersion3, let's say in db schema 26, there will certainly be some migration work to do. If you use MXCryptoVersionLast here, the db will move to MXCryptoVersion3 earlier than expected, specially if someone needs to migrate from schema 15 to 26.